client/reducer/sort.js

```diff
@@ -0,0 +1,33 @@
+export default function sortReducer (state = {}, action) {
+  switch (action.type) {
+    case 'SORT_AZ':
+      return {
+        ...state,
+        sortOrder: 'AZ'
+      }
+
+    case 'SORT_ZA':
+      return {
+        ...state,
+        sortOrder: 'ZA'
+      }
+
+    case 'SORT_BEST':
```

Comment:
Would prefer these be left out of the current branch/PR as they are related to different tickets.